specification/billing/resource-manager/Microsoft.Billing/preview/2018-03-01-preview/examples/BillingAccountsGet.json

```diff
@@ -0,0 +1,17 @@
+{
+  "parameters": {
```

Comment:
Required parameter "get" is missing, see the [log](https://travis-ci.org/Azure/azure-rest-api-specs/jobs/344412418).

Reply:
Fixed.